common/components/utils/truncate.js

```diff
@@ -14,8 +16,8 @@ class Truncate {
 
   static arrayT(arr: Array<string>, length: number): Array<string> {
     if (arr.length > length) {
-     let result: Array<string> = arr.slice(0, length-1);
-     result.push(arr.length - 3 + " more");
+     let result: Array<string> = _.take(arr, length - 1);
```

Comment:
Was array.slice causing errors, or is there a reason not to use a native method over lodash take?

Reply:
It was returning off by one results, which I confirmed with an updated unit test.